cms/src/main/java/uk/nhs/digital/apispecs/swagger/IsAnyTrueHelper.java

```diff
@@ -0,0 +1,39 @@
+package uk.nhs.digital.apispecs.swagger;
+
+import com.github.jknack.handlebars.Helper;
+import com.github.jknack.handlebars.Options;
+import org.apache.commons.lang.Validate;
+
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.Optional;
+
+public class IsAnyTrueHelper implements Helper<String> {
+
+    public static final IsAnyTrueHelper INSTANCE = new IsAnyTrueHelper();
+
+    public static final String NAME = "isAnyTrue";
+
+    @Override public Object apply(final String items, final Options options) throws IOException {
+
+        Validate.notEmpty(options.params,"One or more input parameters");
+        Validate.noNullElements(options.params,"One or more input parameters");
+        Validate.allElementsOfType(Arrays.asList(options.params), Boolean.class,"Required parameter type is Boolean");
+
+        final Options.Buffer buffer = options.buffer();
+
+        if (hasParams(options)) {
+            buffer.append(options.fn());
+        } else {
+            buffer.append(options.inverse());
+        }
+
+        return buffer;
+    }
+
+    private boolean hasParams(final Options options) {
+        return Arrays.stream(Optional.ofNullable(options).get().params)
```

Comment:
The use of `Optional` is redundant here.

We receive options from Handlebars and we can rely on the framework that it can do the right thing. Besides, there are calls to member field of `options` prior to this method that would fail much earlier before this method would be called. Also, as written, the call to `get()` would fail if `options` were `null`.

Reply:
Fixed